cypress/e2e/widgets-landing.cy.ts

```diff
@@ -8,6 +10,48 @@ function resetToDefaultLayout() {
     .click();
 }
 
+const getLayout = () => {
+  return cy
+    .getCookie('cs_jwt')
+    .should('exist')
+    .then((token) => {
+      if (!token || !token.value) {
+        throw new Error('JWT cookie not found.');
+      }
+      return cy
+        .request({
```

Comment:
I am curious as to why we have to do this. Chrome injects auth headers to all requests coming to the same origin.

Reply:
I was seeing a `401 Unauthorized` without adding the auth header here. I removed this function when using the intercept as you suggested below.